### PR TITLE
Add FTMS bikes to sdk

### PR DESCRIPTION
**Pull request by bryanoostra**

### CODE REVIEW

File: domain/src/main/kotlin/com/nohrd/bike/domain/Cadence.kt [hunk 1]
```diff
@@ -0,0 +1,3 @@
+package com.nohrd.bike.domain
+
+public data class Cadence(val value: Double)
```
Comment (nhaarman) [inline domain/src/main/kotlin/com/nohrd/bike/domain/Cadence.kt:3]: Perhaps rename `value` to include the unit? `revolutionsPerMinute`?

File: sdkv2-ble/src/main/kotlin/com/nohrd/bike/sdkv2/ble/CustomNohrdService.kt [hunk 2]
```diff
@@ -0,0 +1,14 @@
+package com.nohrd.bike.sdkv2.ble
+
+import java.util.UUID
+
+/**
+ * Describes the Custom Service used by the Nohrd Bike v2
+ */
+internal object CustomNohrdService {
```
Comment (nhaarman) [inline sdkv2-ble/src/main/kotlin/com/nohrd/bike/sdkv2/ble/CustomNohrdService.kt:8]: These are unused?

Comment (nhaarman) [reply sdkv2-ble/src/main/kotlin/com/nohrd/bike/sdkv2/ble/CustomNohrdService.kt:8]: Or should probably not be `internal` 🤔

**Review by nhaarman — COMMENTED** [review]
Looks good 👍 

I think the `com.nohrd.bike.domain` package can be renamed to just `com.nohrd.bike` 👍

**Comment by nhaarman** [comment]
Could you also update the main README, explaining the different sdk versions?

File: sdkv2-ble/src/main/kotlin/com/nohrd/bike/sdkv2/ble/CustomNohrdService.kt (hunk 2 above)
Comment (bryanoostra) [reply sdkv2-ble/src/main/kotlin/com/nohrd/bike/sdkv2/ble/CustomNohrdService.kt:8]: yes you're right. I added these last minute

**Review by nhaarman — APPROVED** [review]

**Pull request merged**